packages/components/src/components/entities/actions.ts

```diff
@@ -105,7 +105,7 @@ function initParents(initialParents: string[], selectionKey: string): Promise<Li
                     .then(response => resolve(response))
                     .catch(reason => reject(reason));
             } else {
-                return getSelected(selectionKey)
+                return getSelected(selectionKey, undefined, undefined, undefined, undefined, undefined)
```

Comment:
nit: Why add these `undefined` parameters? They're marked on the interface as optional which should suffice.

Reply:
Just a refactoring artifact, I believe.  I shall remove